Mathlib/CategoryTheory/Limits/Shapes/Multiequalizer.lean

```diff
@@ -377,9 +358,7 @@ def ofι (I : MulticospanIndex.{w} C) (P : C) (ι : ∀ a, P ⟶ I.left a)
         | WalkingMulticospan.right b => ι (I.fstTo b) ≫ I.fst b
       naturality := by
         rintro (_ | _) (_ | _) (_ | _ | _) <;>
-          dsimp <;>
-          simp only [Category.id_comp, Category.comp_id, Functor.map_id,
-            MulticospanIndex.multicospan_obj_left, MulticospanIndex.multicospan_obj_right]
+          dsimp <;> simp
```

Comment:
I noticed that this `dsimp` is unnecessary but removing it significantly slows down the proof.